ios/RNTPrimer.swift

```diff
@@ -0,0 +1,312 @@
+//
+//  Primer.swift
+//  primer-io-react-native
+//
+//  Created by Evangelos on 15/3/22.
+//
+
+import Foundation
+import PrimerSDK
+
+@objc
+enum PrimerEvents: Int, CaseIterable {
+    case onClientTokenCallback = 0
+    case onClientSessionActions
+    case onTokenizeSuccessCallback
+    case onResumeSuccess
+    case onCheckoutDismissed
+    case onError
+    
+    case detectImplementedRNCallbacks
+    
+    var stringValue: String {
+        switch self {
+        case .onClientTokenCallback:
+            return "onClientTokenCallback"
+        case .onClientSessionActions:
+            return "onClientSessionActions"
+        case .onTokenizeSuccessCallback:
+            return "onTokenizeSuccessCallback"
+        case .onResumeSuccess:
+            return "onResumeSuccess"
+        case .onCheckoutDismissed:
+            return "onCheckoutDismissed"
+        case .onError:
+            return "onError"
+        case .detectImplementedRNCallbacks:
+            return "detectImplementedRNCallbacks"
+        }
```

Comment:
```suggestion
     return String(describing: self)
```

Reply:
Have you tested this? I believe that it wouldn't return the case, but the enum's name.

Reply:
I did, it returns the `case` name which matches in this scenario all the literal strings that we have
![Screen Shot 2022-03-23 at 09 44 15](https://user-images.githubusercontent.com/3584008/159658785-619f9bbd-f6f8-40bb-b390-782f81f143f7.png)
.